teaser/project.py

```diff
@@ -1079,6 +1079,15 @@ def name(self):
 
     @name.setter
     def name(self, value):
-
-        self._name = value
-        self.modelica_project = value
+        if isinstance(value, str):
+            regex = re.compile('[^a-zA-z0-9]')
+            self._name = regex.sub('', value)
```

Comment:
@PRemmen, it seems as `self.modelica_project` is a duplicate of `self.name`. Would deleting this attribute require adapting the xsd schema and the bindings?